src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -124,6 +125,39 @@ public override async Task<IReadOnlyList<PackageSpec>> GetPackageSpecsAsync(Depe
             return new[] { packageSpec };
         }
 
+        private async Task<bool> IsCentralPackageManagementVersionsEnabledAsync()
+        {
+            return MSBuildStringUtility.IsTrue(await _vsProjectAdapter.GetPropertyValueAsync(nameof(ProjectBuildProperties.ManagePackageVersionsCentrally)));
```

Comment:
I followed the pattern used for other properties. Is the suggestion to make it constant?

Reply:
The pattern that we are using is the following: 

```cs
public static class ProjectBuildProperties {
        public const string ManagePackageVersionsCentrally= nameof(ManagePackageVersionsCentrally);
}
```

Then later you just use: 

```
            return MSBuildStringUtility.IsTrue(await _vsProjectAdapter.GetPropertyValueAsync(ProjectBuildProperties.ManagePackageVersionsCentrally));

```

Reply:
I disagree about the refactoring concern. 
I think it's not a concern. The idea with nameof is to avoid typos everywhere and keep things consistent. 

An engineer should be aware that changing a property can technically break the scenario.

Reply:
> The idea with nameof is to avoid typos everywhere and keep things consistent.

Avoiding typos and keeping things consistent is achieved by having the single property that all code re-uses, rather than defining their own local value. `nameof` doesn't solve that.

> An engineer should be aware that changing a property can technically break the scenario.

I find that really scary. Renaming a variable should never break code.

Reply:
> I find that really scary. Renaming a variable should never break code.

I have never seen issues caused because of that. I think it makes an assumption that people don't pay any attention to what's being committed :) 

It's common in many repos: https://github.com/dotnet/project-system/blob/master/src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/ProjectCapability.cs
https://github.com/dotnet/sdk/blob/master/src/Cli/dotnet/UILanguageOverride.cs

Reply:
Updated with _vsProjectAdapter.GetPropertyValueAsync(ProjectBuildProperties.ManagePackageVersionsCentrally)